dashboard/config/routes.rb

```diff
@@ -309,6 +309,8 @@
   resources :courses, param: 'course_name'
   get '/course/:course_name', to: redirect('/courses/%{course_name}')
 
+  resources :lessons, only: [:show, :edit, :update]
```

Comment:
I would love a link to read about how this works.

Reply:
here is a good starting point: https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default

let me know if you have follow-up questions.

Reply:
the other mysterious thing to read about in this PR is CanCan. https://github.com/CanCanCommunity/cancancan it has to do with the following aspects of this PR:
* `load_and_authorize_resource`
* `can?`
* ability.rb